oct/pages/base/base.py

```diff
@@ -41,31 +39,29 @@ def __init__(self, base: Any) -> None:
         self._base = base
 
     contains: Optional[dict] = None
-    """
-    contains class attribute contains page's webelements. Webelements set
-    is the page specific.
-    Example:
-    contains = {
-        elementName: {
-            "locator": (by, value),
-            "class": Element
-        },
-        ...
-    }
-
-    For pages with dynamically generated content (Search Page, Category Page)
-    specify loadable webelement (such ProductThumb) by adding "is_loaded"
-    parameter to the element's dict as such:
-    contains = {
-        ...
-        elementName: {
-            "locator": (by, value),
-            "class": ProductThumb,
-            "is_loaded": True
-        },
-        ...
-    }
-    """
+    # contains class attribute contains page's webelements. Webelements set
+    # is the page specific.
+    # Example:
+    # contains = {
+    #     elementName: {
+    #         "locator": (by, value),
+    #         "class": Element
+    #     },
+    #     ...
+    # }
+
+    # For pages with dynamically generated content (Search Page, Category Page)
+    # specify loadable webelement (such ProductThumb) by adding "is_loaded"
+    # parameter to the element's dict as such:
+    # contains = {
+    #     ...
+    #     elementName: {
+    #         "locator": (by, value),
+    #         "class": ProductThumb,
+    #         "is_loaded": True
+    #     },
+    #     ...
+    # }
```

Comment:
why?

Reply:
It is preferable to use comments for code description